apps/odyssey-stats/src/widget/modules.tsx

```diff
@@ -64,7 +64,14 @@ const ModuleCard: FunctionComponent< ModuleCardProps > = ( {
 				<>
 					{ ( ! isError || ! canManageModule ) && (
 						<div className="stats-widget-module__value">
-							<ShortenedNumber value={ value } />
+							<span>
+								{ numberFormat( value, {
+									numberFormatOptions: {
+										notation: 'compact',
+										maximumFractionDigits: 1,
+									},
+								} ) }
```

Comment:
We may come back to build a `compact` abstraction into `numberFormat`. So passing `maximumFractionDigits: 1` won't be necessary.